src/SearchBar.jsx

```diff
@@ -0,0 +1,187 @@
+var React           = require('react/addons')
+	,{LinkedStateMixin} = React.addons
```

Comment:
don't use anything from addons, it can cause issues by creating duplicate instances of React. In this case just hook the value up to state manually